src/DynamoDbModel.php

```diff
@@ -12,6 +12,13 @@
 abstract class DynamoDbModel extends Model
 {
     /**
+     * The connection name for the model.
+     *
+     * @var string
+     */
+    protected $connection;
```

Comment:
This variable exists on the Eloquent Model already - are you just defining it here for clarity?

Reply:
At the time of writing, yeah it was the intention. But then, we are not listing all of the properties here anyway (`primaryKey`, `table`, etc.). So I'll remove it.